aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -419,15 +420,14 @@ c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>> PackedConvWeightsOnednn<
     dims_giohw = with_groups ? ideep::utils::group_dims(dims_iohw, groups) : dims_iohw;
     std::vector<int64_t> perms(dims_giohw.size(), 0); // for permutation of weight
     std::iota(perms.begin(), perms.end(), 0);
-    w_desc = w_desc.transpose(with_groups, with_groups + 1);
```

Comment:
Explain why this line is removed?

Reply:
A flag is added in ideep to control the weight layout to be onednn-style or pytorch-style (One is the transpose of the other): https://github.com/intel/ideep/pull/146
Now we set the flag to have the onednn-style layout, so we do not need to transpose it here anymore.